packages/eslint-plugin-query/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@tanstack/eslint-plugin-query",
-  "version": "4.32.5",
+  "version": "4.32.6",
```

this will be incremented automatically by our build script, so please revert that:

```suggestion
  "version": "4.32.5",
```